components/MembershipsWithData.js

```diff
@@ -4,12 +4,21 @@ import { gql } from '@apollo/client';
 import { graphql } from '@apollo/client/react/hoc';
 import { Button } from 'react-bootstrap';
 import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
 
+import Container from './Container';
 import Error from './Error';
 import Membership from './Membership';
 
 const MEMBERSHIPS_PER_PAGE = 10;
 
+const MembersContainer = styled.div`
+  .loadMoreBtn {
```

Comment:
Since we're in control of this one, maybe we can remove the class? A nice change would be to replace the `Button` by a `StyledButton`

Reply:
@Betree : Good idea. I've checked in the change; here's the new look; let me know if you see any other issues. 

![image](https://user-images.githubusercontent.com/12435965/100911149-b65f6f80-3483-11eb-80c9-2db2c86bb712.png)